dotnet-desktop-guide/net/winforms/wfdev-diagnostics/wfac001.md

```diff
@@ -2,10 +2,19 @@
 title: WFAC001 error
 description: Learn about the Windows Forms code that generate compile-time error WFAC001.
 ms.date: 11/22/2023
+f1_keywords:
+  - "WFAC001"
+helpviewer_keywords:
+  - "WFAC001"
+# NOTE: This error/warning is part of the old diagnostic identifiers. It's not being maintained.
```

Comment:
This was replaced in NET9

Reply:
There's a note after the introduction indicating that this was replaced by a different error.